spinalcordtoolbox/types.py

```diff
@@ -564,15 +590,25 @@ def compute_vertebral_distribution(self, disks_levels, label_reference='C1'):
 
                     if current_label in ['PMJ', 'PMG']:
                         if next_label in self.index_disk:
-                            self.dist_points_rel[i] = - (self.dist_points[i] - self.dist_points[self.index_disk[next_label]]) / abs(self.dist_points[self.index_disk[next_label]] - self.dist_points[self.index_disk[current_label]])
+                            self.dist_points_rel[i] = -(
+                                self.dist_points[i] - self.dist_points[self.index_disk[next_label]]
```

Comment:
I was tempted to replace this instance of `-(x-y)` with the equivalent `(y-x)`, but I didn't dare.